internal/backend/rename_unix.go

```diff
@@ -0,0 +1,21 @@
+//go:build !windows
+// +build !windows
```

Comment:
Hmm now that we only need to support Go 1.18 and 1.19, we can get rid of `+build` (`go fix` should do it automatically I think).